lib/objects/objectsInMemFileDB.js

```diff
@@ -302,19 +307,7 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
 
     _readFile(id, name, options, callback) {
         try {
-            if (!this.fileOptions[id]) {
-                const location = path.join(this.objectsDir, id, '_data.json');
-                if (fs.existsSync(location)) {
-                    try {
-                        this.fileOptions[id] = JSON.parse(fs.readFileSync(location, 'binary'));
```

Comment:
Uh-oh! Line 277 was reading `utf8`, this one expected `binary`. Since both are JSON files, I guess its fine, but better double-check.

Reply:
not existing anymore ?!